pandas/tests/io/formats/test_to_csv.py

```diff
@@ -561,3 +563,19 @@ def test_to_csv_compression(self, compression_only,
             result = pd.read_csv(path, index_col=0,
                                  compression=read_compression)
             tm.assert_frame_equal(result, df)
+
+    def test_to_csv_na_rep_long_string(self, capsys):
+        # see gh-25099
+        df = pd.DataFrame({"c": [float('nan')] * 3})
+        df = df.astype("Int64")
```

Comment:
can you parametrize this function on ``float`` and ``Ing64``

Reply:
Good idea. Done!